examples/plot_01_getting_started.py

```diff
@@ -17,6 +17,8 @@
 ==========================================================
 """
 
+# ruff: noqa: E402
```

Comment:
I would suggest to add the rule in the `pyproject.toml` to not have it in the example.

As an example: https://github.com/scikit-learn/scikit-learn/blob/main/pyproject.toml#L171-L174

Reply:
It is less invasive and centralized.

Reply:
Its unfortunately not at the same level (we have chosen a mono-repo/multi-package architecture).
The configuration in `skore/` will not impact `ruff` on the `examples/` directory.
The examples are at the root level, and the python packages in dedicated sub-folders.

I wouldn't like to put examples in the `skore/` directory.

Reply:
Arff, in this case, you can have a `ruff.toml` in the root.

Reply:
I made an in-between: i put the configuration in `examples/ruff.toml` to allow each package to define its own configuration without side-effect.